app/src/main/java/org/schabi/newpipe/util/external_communication/TextLinkifier.java

```diff
@@ -33,7 +33,8 @@
 public final class TextLinkifier {
     public static final String TAG = TextLinkifier.class.getSimpleName();
 
-    private static final Pattern HASHTAGS_PATTERN = Pattern.compile("(#[A-Za-z0-9_]+)");
+    private static final Pattern HASHTAGS_PATTERN =
+            Pattern.compile("(#[A-Za-z0-9_\\u0080-\\uFFFF]+)");
```

Comment:
Could you add an comment on what `\\u0080-\\uFFFF` is? From the looks of it, it is all unicode characters except the first 80

Reply:
Outdated